Gemfile

```diff
@@ -3,4 +3,5 @@ source "https://rubygems.org"
 gem 'kitchen-docker', '>= 2.9'
 gem 'kitchen-salt', '>= 0.6.0'
 gem 'kitchen-inspec', '>= 1.1'
+gem 'inspec'
```

Comment:
Is this necessary?  It's pulled automatically as a dependency of `kitchen-inspec`:

* https://rubygems.org/gems/kitchen-inspec/versions/1.1.0

Reply:
No, you're right. A local test. Removing.

Reply:
Fixed